src/components/show-edit-pod/show-edit-pod.js

```diff
@@ -75,34 +70,36 @@ class ShowEditPod extends React.Component {
     }
   }
 
-  control = 'props';
+  deleteButton() {
+    const label = this.props.deleteText || I18n.t('actions.delete', { defaultValue: 'Delete' });
 
-  get stateControlled() {
-    return this.control === 'state';
-  }
-
-  get mainClasses() {
-    return classNames(
-      'carbon-show-edit-pod',
-      this.props.className
-    );
-  }
+    if (isClassic(this.props.theme)) {
+      return (
+        <StyledLink
```

Comment:
Should a StyledLink have a tooltip and/or onKeyDown() for accessibility reasons?

Reply:
it's just a Link Component with custom styles, and it's already implemented in that Component.